app/views/admin/members/_form.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= form_with(model: [:admin, member], builder: SpectreFormBuilder) do |form| %>
-  <%= render partial: 'fields', locals: { form: form, show_password_inputs: true } %>
+  <%= render partial: 'fields', locals: { form: form, show_password_inputs: true, librarian_view: true } %>
```

Comment:
Maybe we could rename this param to `show_pronunciation_inputs` to mirror the `show_password_inputs` param?

Reply:
I use `librarian_view` because I guess there could be more fields which only show in librarian view but not in member view. In that case we can reuse this variable in the future.
What do you think? If the possibility of that is low, I will change the name like you suggested.

Reply:
That's a good point. Let's leave the name as you have it then.